.yarnrc.yml

```diff
@@ -1,3 +0,0 @@
-nodeLinker: node-modules
```

Comment:
dont delete this file

Reply:
@maxiroellplenty It's fine to delete it, we don't need it anymore for the workflows. But if you do, please also remove it from the git index via `git rm --cached .yarnrc.yml`.

Reply:
ah ok, I see 👍